sdk/program/src/vote/state/mod.rs

```diff
@@ -151,6 +151,10 @@ impl VoteStateUpdate {
     pub fn slots(&self) -> Vec<Slot> {
         self.lockouts.iter().map(|lockout| lockout.slot).collect()
     }
+
+    pub fn compact(self) -> Option<CompactVoteStateUpdate> {
+        CompactVoteStateUpdate::new(self.lockouts, self.root, self.hash, self.timestamp)
+    }
```

Comment:
Can we remove the #![allow(clippy::integer_arithmetic)] at the top of the file now?